.travis.yml

```diff
@@ -24,7 +24,7 @@ before_script:
 script:
     # Run pycodestyle on all .py files in all subfolders
     - black --check .
-    - find . -type d -o ! -name versioneer.py ! -name test_*.py ! -name _version.py -name \*.py -exec pydocstyle --convention=numpy --add-ignore=D105,D202 {} +
+    - find . -type d -o ! -name versioneer.py ! -name test_*.py ! -name DummyClient.py ! -name _version.py -name \*.py -exec pydocstyle --convention=numpy --add-ignore=D105,D202 {} +
```

Comment:
this excludes `DummyClient.py` from a check by pydocstyle, a wonderfully annoying tool to nag about grammar in your docstrings:
```
./tests/DummyClient.py:1 at module level:

        D100: Missing docstring in public module

./tests/DummyClient.py:26 in public function `stopwatch`:

        D200: One-line docstring should fit on one line with quotes (found 2)

./tests/DummyClient.py:60 in public class `DummyClient`:

        D400: First line should end with a period (not ',')

./tests/DummyClient.py:67 in public method `__init__`:

        D401: First line should be in imperative mood (perhaps 'Set', not 'Sets')

./tests/DummyClient.py:85 in public method `register_start`:

        D208: Docstring is over-indented

./tests/DummyClient.py:85 in public method `register_start`:

        D214: Section is over-indented ('Parameters')

./tests/DummyClient.py:85 in public method `register_start`:

        D215: Section underline is over-indented (in section 'Parameters')

./tests/DummyClient.py:85 in public method `register_start`:

        D214: Section is over-indented ('Raises')

./tests/DummyClient.py:85 in public method `register_start`:

        D215: Section underline is over-indented (in section 'Raises')

./tests/DummyClient.py:85 in public method `register_start`:

        D214: Section is over-indented ('Parameters')

./tests/DummyClient.py:85 in public method `register_start`:

        D215: Section underline is over-indented (in section 'Parameters')

./tests/DummyClient.py:85 in public method `register_start`:

        D214: Section is over-indented ('Raises')

./tests/DummyClient.py:85 in public method `register_start`:

        D215: Section underline is over-indented (in section 'Raises')

./tests/DummyClient.py:158 in public method `register_config`:

        D401: First line should be in imperative mood (perhaps 'Register', not 'Registers')

./tests/DummyClient.py:285 in public method `request_state`:

        D400: First line should end with a period (not 'D')

./tests/DummyClient.py:355 in public method `update_datasets`:

        D401: First line should be in imperative mood (perhaps 'Query', not 'Queries')

./tests/DummyClient.py:427 in public class `DummyClientLocust`:

        D101: Missing docstring in public class
```